src/core_plugins/input_control_vis/index.js

```diff
@@ -0,0 +1,9 @@
+export default function (kibana) {
```

Comment:
Great idea. I have split the editor into two tabs; `Controls` and `Options`

The idea behind disabling "Update kibana filters on each change" by default is that only submitting all of the filters at a single instant puts the least amount of stress on the cluster. The default is designed for the usecase where the Panel contains many controls. Users can fiddle with several filters and then apply them all at once - so the dashboard only has to update on time instead of each time a control is modified.